Description: Fix unit test that fails to compile on i386
 The value is too large for architectures with 32-bit longs, resulting
 in a "narrowing conversion" compiler error.
Author: Doug Torrance <dtorrance@piedmont.edu>
Bug: https://github.com/Macaulay2/M2/pull/1540
Forwarded: https://github.com/Macaulay2/M2/pull/1560
Last-Update: 2020-10-27

--- a/M2/Macaulay2/e/unit-tests/basics-test.cpp
+++ b/M2/Macaulay2/e/unit-tests/basics-test.cpp
@@ -56,9 +56,9 @@
 
 TEST(Util, m2array2stdvec_big)
 {
-  std::vector<long> a{-1453853049583, 3, 6, 4, -2};
+  std::vector<long long> a{-1453853049583, 3, 6, 4, -2};
   M2_arrayint b = stdvector_to_M2_arrayint(a);
-  std::vector<long> c = M2_arrayint_to_stdvector<long>(b);
+  std::vector<long long> c = M2_arrayint_to_stdvector<long long>(b);
   EXPECT_FALSE(a == c);
 }
 
